### PR TITLE
Cnh/intregate UI core from npm

### LINKED ISSUE #7158: Prepare UI-core integration
### Description and goal

Preparing the integration of the UI-core components into OSRD

### Acceptance criteria

- [x] Publication of osrd-project on npm : https://www.npmjs.com/search?q=osrd
- [x] Release script for osrd-project on npm
- [ ] Correct CSS style conflicts #7162

### PR DESCRIPTION
**Pull request by kmer2016**
Testing the integration of ui-core into osrd/front. 

:warning: Do not merge before removing the last commit, which is only there to test the integration of the ui-core input into an osrd form (rollingstockEditor). 
image

To handle the integration properly, we need to reset some styles. This is due to the base style of Tailwind that comes with ui-core, which sets the default style for these HTML elements. In particular : 

- the display of **img** and **svg** is set to display:block, which positions them above or below their sibling elements. (i[ssue reported here](https://github.com/tailwindlabs/tailwindcss/issues/942)).

- the height of the img is set to auto, which causes the img to take up all the available space of the parent element even if you define a height in the style ([issue reported here](https://github.com/tailwindlabs/tailwindcss/pull/7742#issuecomment-1061332148)). 

- The **button** has a background-color transparent as mentionned [here](https://github.com/tailwindlabs/tailwindcss/issues/6602)

### CODE REVIEW

**Review by SharglutDev — COMMENTED** [review]
When I try to focus an input with tab, I get this : 

image

Is that normal ?

File: front/src/styles/styles.scss [hunk 1]
```diff
@@ -12,3 +12,4 @@
 @import 'scss/applications/operationalStudies';
 @import 'scss/applications/rollingStockEditor/rollingStockForm';
 @import 'scss/applications/stdcm';
+@import 'scss/_uiCoreIintegration.scss';
```
Comment (clarani) [inline front/src/styles/styles.scss]: You forgot an empty line here :)

**Review by clarani — CHANGES_REQUESTED** [review]
Great PR, I spotted some bugs:

- display is broken if I click on the arrow next to the eye in the scenario page :eye: 
image

- the display of the simulation parameters is not correct, same for the import tab and the stdcm page
image
image
image

- the rolling stock image is very small in the rolling stock selector
image

- the label is transparent on the editor
image

- the selector modal is broken in the editor
image

- the popover to search the map is too large
image

- the popover to select the data layer is also broken in the main map
image

File: front/src/styles/scss/_uiCoreIntegration.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+.layer-modal-img  {
+  height: 20px;
+}
+
+//https://github.com/tailwindlabs/tailwindcss/issues/6602
+.error-box button {
+  background-color: rgb(239, 239, 239);
+  color: black;
+}
```
Comment (clarani) [inline front/src/styles/scss/_uiCoreIntegration.scss]: Missing new line

**Review by clarani — COMMENTED** [review]
The interface is perfect now, thanks, I only noticed that the input does not seem to work and you forgot a new line :+1:

**Review by clarani — APPROVED** [review]
LGTM :white_check_mark: You can merge after you have done these few things:
- [x] fix the missing new line
- [x] remove the commit which test the new input component

The bugs related to the input will be fixed later in osrd-ui

**Review by clarani — APPROVED** [review]

**Pull request merged**